python/examples/integrations/BigQuery_Example.ipynb

```diff
@@ -0,0 +1,411 @@
+     },
+     "execution_count": 19,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "ProfileSummaryReport(target_view=profile_view).report()"
+   ],
+   "metadata": {
+    "collapsed": false,
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "This report is very useful, since it shows us with a bit more detail how does the distributions behave, and also some core metrics that we will sometimes need to refer to quickly."
```

Comment:
```suggestion
    "This report can be very useful, since it shows us in a bit more detail the characteristics of the distributions, and also lists some of the core metric names that we can refer to quickly."
```